cumulusci/robotframework/pageobjects/ObjectManagerPageObject.py

```diff
@@ -50,6 +79,7 @@ def _is_current_page(self):
 
     @capture_screenshot_on_error
     def switch_tab_to(self, tab):
+        """Switches tab to a specified tab"""
```

Comment:
This document isn't quite enough if all you have to go on is the description (ie: if you're reading it on a web page of keyword definitions). You should describe what the "tab" argument is. Is it a number? The label of the tab? A tab locator? It would also be good to include an example. 

You can include an example by using a vertical bar at the start of a line to create block of preformatted code. For example:
```
""" blah blah blah

Example:

| switch tab to  Fields & Relationships
```
You can also use a | to create a table if the line also ends with a |

```
| switch to tab  | Fields & Relationships |
```

Reply:
updated